.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**Leetcode problem**
+<!-- Link to example problem on leetcode where the bug manifests if applicable. If not just put N/A -->
+
+**To Reproduce**
+<!-- Steps / command to reproduce the behavior. Example  `cargo leet gen https://leetcode.com/problems/all-nodes-distance-k-in-binary-tree/`-->
+``
+
+**Error Message**
+<!-- If applicable please provide a copy of the error received or a screenshot of the error -->
+```
+```
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. Not necessary if error was received. -->
+
+**Environment Info:**
+<!-- Use `cargo version` at command line to get version of cargo. e.g. "cargo 1.70.0 (ec8a8a0ca 2023-04-25)" -->
+ - Cargo Version:
```

Comment:
OS ??

Reply:
I'm not sure OS matters. None of what we use is OS dependent to be knowledge. Don't want to give people unnecessary work